ccd-definition/AuthorisationCaseState/CareSupervision/AuthorisationCaseState-nonprod.json

```diff
@@ -34,6 +34,13 @@
     "UserRole": "caseworker-publiclaw-judiciary",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseStateID": "ISSUE_RESOLUTION",
```

Comment:
I guess we dont need it now but probably better to add it and delete it later when we remove the state